iaso/gpkg/import_gpkg.py

```diff
@@ -153,6 +186,55 @@ def get_ref(inst: Union[OrgUnit, Group]) -> str:
     return inst.source_ref if inst.source_ref else f"iaso#{inst.pk}"
 
 
+def validate_required_property(props: Dict[str, str], property_name: str, orgunit_name: str = "") -> str:
+    """Check if a required property exists and is not empty
+    Args:
+        props: Dictionary of properties
+        property_name: Name of the property to check
+        orgunit_name: Name of orgunit for error message (optional)
+    Returns:
+        The property value
+    Raises:
+        ValueError if:
+        - property column doesn't exist in props
+        - property value is None
+        - property value is empty string
+        - property value is only whitespace
+    """
+    if property_name not in props:
+        raise ValueError(f"Column '{property_name}' is required but missing from GPKG")
+
+    value = props[property_name]
+    if value is None:
+        raise ValueError(f"Column '{property_name}' cannot be null")
+
+    if not isinstance(value, str):
+        raise ValueError(f"Column '{property_name}' must be a string, got {type(value)}")
+
+    if value.strip() == "":
+        raise ValueError(f"Column '{property_name}' cannot be empty or blank")
+
+    return value.strip()
+
+
+def validate_property(props: Dict[str, str], property_name: str, orgunit_name: str = "") -> Optional[str]:
+    """Check if a property exists and validate it's not empty if present
+    Args:
+        props: Dictionary of properties
+        property_name: Name of the property to check
+        orgunit_name: Name of orgunit for error message (optional)
+    Returns:
+        The property value if present and non-empty, None if property doesn't exist or is empty
+    """
+    if property_name not in props:
+        return None
+
+    value = props[property_name]
+    if value and value.strip():  # Only return non-empty values
```

Comment:
I think I would prefer value.strip() == "" to be totally sure of what that test does